locale/zh_CN.po

```diff
@@ -5090,7 +5091,8 @@ msgstr "WebKit v1 不支持检索 JavaScript 脚本输出"
 msgid ""
 "GTK+ installed on this machine is too old to support screen compositing, "
 "please install GTK+ 2.12 or later."
-msgstr "本机安装的GTK+版本过低，不支持屏幕组合，请安装GTK+2.12或者更新版本。"
+msgstr ""
```

Comment:
Will this empty string get an error or it will be ignored?

Reply:
![Screen Shot 2022-07-11 at 10 41 24](https://user-images.githubusercontent.com/82932766/178178971-f265877d-8f88-4de3-bbb4-300a787a7851.png)

Reply:
5094 will be just an empty line. not even a quotation mark will be seen.